<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaceOrderForm</class>
 <widget class="QWidget" name="PlaceOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>平仓</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合约：</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_contract">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>买卖：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_buy">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>买</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sell">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>卖</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_vol">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数量：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_price_type">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>限价</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>对价</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>价格类型：</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>开仓</string>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>plainTextEdit_contract</zorder>
  <zorder>label_2</zorder>
  <zorder>radioButton_buy</zorder>
  <zorder>radioButton_sell</zorder>
  <zorder>spinBox_vol</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBox_price_type</zorder>
  <zorder>label_4</zorder>
  <zorder>doubleSpinBox_price</zorder>
  <zorder>label_5</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>plainTextEdit_contract</tabstop>
  <tabstop>radioButton_buy</tabstop>
  <tabstop>radioButton_sell</tabstop>
  <tabstop>spinBox_vol</tabstop>
  <tabstop>comboBox_price_type</tabstop>
  <tabstop>doubleSpinBox_price</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
